src/component/admin/project-roles/ProjectRoleForm/EnvironmentPermissionAccordion/EnvironmentPermissionAccordion.tsx

```diff
@@ -128,6 +128,7 @@ const EnvironmentPermissionAccordion = ({
                             text={environment.name}
                             className={styles.header}
                             maxWidth="120"
+                            maxLength={25}
```

Comment:
Do we need both `maxWidth` and `maxLength`? Maybe `maxLength` is enough. Probably out of scope though.

Reply:
We can probably get away with setting a fixed maxWidth, but in some cases it might be good to choose. I think this whole component would benefit from a review in due time.